lib/bn.js

```diff
@@ -459,6 +460,11 @@ BN.prototype.toJSON = function toJSON() {
   return this.toString(16);
 };
 
+BN.prototype.toBuffer = function toBuffer(endian, length) {
```

Comment:
Yeah, it will require a bit of rewriting of code below, but I think it should be possible. Are you interested in working on this?